run_tests.sh

```diff
@@ -1,15 +1,18 @@
 #!/usr/bin/env bash
 
+which nosetests || (echo "[ERROR] nosetests framework is needen to run the tests" && exit 1)
+which clang || (echo "[ERROR] clang required to run functional tests" && exit 1)
+
 # setup environment variables, temporary directories
-. tests/test_env_setup.sh
+. tests/test_env_setup.sh || exit 1
 
 # build the package used for the tests
-./build_package.py -o $TEST_CODECHECKER_PACKAGE_DIR -v
+./build_package.py -o $TEST_CODECHECKER_PACKAGE_DIR -v || exit 1
 
 # run all the tests
 nosetests tests/test_packages/
 
 # cleanup the environment variables and temporary
 # directories
-clean_tests.sh
+./clean_tests.sh
```

Comment:
This file should be placed in `tests` directory, like `test_env_setup.sh`. As this is in the root like `run_tests.sh`, it seems that there is some reason for the user to run it manually.